docs/csharp/language-reference/compiler-messages/cs8210.md

```diff
@@ -0,0 +1,35 @@
+---
+description: "Compiler Error CS8210"
+title: "Compiler Error CS8210"
+ms.date: 10/26/2022
+f1_keywords:
+  - "CS8210"
+helpviewer_keywords:
+  - "CS8210"
+---
+# Compiler Error CS8210
+
+A tuple may not contain a value of type 'void'.
+
+## Example
+
+ The following sample generates CS8210:
+
+```csharp
+// CS8210.cs
+void Method()
+{
+}
+
+void Test()
+{
+    var x = ("something", Method());
+}
+```
+
+Your method returns void, so that method doesn't return a value. You can't use a method that returns void for a data member of a tuple.
```

Comment:
We should code fence `void`:

```suggestion
Your method returns `void`, so that method doesn't return a value. You can't use a method that returns `void` for a data member of a tuple.
```